pkg/config/config.go

```diff
@@ -145,6 +150,20 @@ func (cfg *Config) BindFlags() {
 		"Specific namespace the service controller will watch for object creation from CRD. "+
 			" By default it will listen to all namespaces",
 	)
+	flag.IntVar(
+		&cfg.DefaultDriftRemediationPeriod, flagDefaultDriftRemediationPerdiod,
+		60,
+		"The default duration, in seconds, to wait before attempting to remediate drift in custom resources. "+
+			"This value is used if no resource-specific period or requeue on success period has been specified. "+
+			"Default is 60 seconds.",
+	)
+	flag.StringVar(
+		&cfg.DriftRemediationConfigurationJSON, flagDriftRemediationConfigurationJSON,
```

Comment:
How about we call this flag `--reconcile-resource-resync-seconds` and make it a [`flag.StringSliceVar`](https://pkg.go.dev/github.com/spf13/pflag#StringSliceVar). Then split each string by the `=` sign to form an eventual `map[string]int`.

That way, the user could specify multiple overrides with multiple invocations of the flag, like so:

```bash
./controller --reconcile-resource-resync-seconds "Function=30" \
  --reconcile-resource-resync-seconds "Alias=120"
```

and there would be no annoying JSON serialization/deserialization to do in a command-line flag.

Reply:
I like it!